media/js/base/protocol/m24-navigation.es6.js

```diff
@@ -20,6 +22,22 @@ const _wideBreakpoint = '768px';
 const _tallBreakpoint = '600px';
 let _mqLargeNav;
 const _viewport = document.getElementsByTagName('html')[0];
+let _enableKeyboardFcousTrap = false;
+let _navigationKeyboardFocusTrapCleanUp = null;
+
+/**
+ * Enable the keyboard focus trapfor navigation
+ */
+MzpNavigation.enableKeyboardFcousTrap = () => {
```

Comment:
Nit: `Focus` is incorrectly spelled here, and is also present in other variables like `_enableKeyboardFcousTrap`
```suggestion
MzpNavigation.enableKeyboardFocusTrap = () => {
```